packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -1446,7 +1458,10 @@ def get_signature(
         return signature_bytes
 
     def send_raw_transaction(
-        self, transaction: RawTransaction
+        self,
+        transaction: RawTransaction,
+        use_flashbots: bool = True,
```

Comment:
I think this should be False.

Reply:
Addressed on 7f3d8d469c4609f63187de244688c047bbd2a5eb.